src/controllers/credential-status.ts

```diff
@@ -496,6 +498,12 @@ export class CredentialStatusController {
 
 		// handle error
 		if (!result.isEmpty()) {
+			const message = result.array().pop()?.msg;
+			eventTracker.emit('notify', {
+				message: `createUnencryptedStatusList. Request validation error: ${message}`,
+				severity: 'error',
```

Comment:
Validation errors are user-originated, which is not an error per se.

We should probably lower the severity to `info`.